test/tests/jobs/GetJobsTest.cpp

```diff
@@ -0,0 +1,131 @@
+#include <test/lib/BedrockTester.h>
+#include <time.h>
+
+// Get a unix timestamp from one of or sqlite date strings.
+time_t stringToUnixTimestamp(const string& timestamp) {
+    struct tm time;
+    strptime(timestamp.c_str(), "%Y-%m-%d %H:%M:%S", &time);
+    return mktime(&time);
+}
+
+// Get the difference in seconds between a and b
+uint64_t absoluteDiff(time_t a, time_t b) {
+    if (a > b) {
+        return a - b;
+    }
+    return b - a;
```

Comment:
couldn't this just be `(a -b) * -1`?

Reply:
They're unsigned, so `(1 - 2)` = `(giant positive number) * -1` = `???`

Reply:
`time_t` is allowed to be signed or unsigned, really, but the return type is unsigned and this works with either case.

Reply:
Also, how is `(a - b) * -1)` better than `b - a`?

Reply:
ah okay